pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -24,19 +30,19 @@ This section describes how to upgrade your Kommander Management cluster and all
   Download the Kommander application definitions:
 
   ```bash
-  wget "https://downloads.d2iq.com/dkp/v2.2.0/kommander-applications_v2.2.0.tar.gz"
+  wget "https://downloads.d2iq.com/dkp/${VERSION}/kommander-applications-${VERSION}.tar.gz"
   ```
   
   Download the Kommander charts bundle:
 
   ```bash
-  wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-kommander-charts-bundle-v2.2.0.tar.gz"
+  wget "https://downloads.d2iq.com/dkp/${VERSION}/dkp-kommander-charts-bundle-${VERSION}.tar.gz"
   ```
   
   If you have any DKP Catalog Applications, download the DKP Catalog Application charts bundle:
 
   ```bash
-  wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-catalog-applications-charts-bundle-v2.2.0.tar.gz"
+  wget "https://downloads.d2iq.com/dkp/${VERSION}/dkp-catalog-applications-charts-bundle-${VERSION}.tar.gz"
```

Comment:
is this really what we want to do?? I think it's deviating from what we've done in other locations, specifically Konvoy.

Reply:
I'm aligned with @cneth's opinion on it, but consistency is important. We do this in a lot of places in the Kommander docs already. I'm happy to go either way!

Reply:
I think we should keep `https://downloads.d2iq.com/dkp/v2.2.0/` to maintain consistency with the rest of the of Konvoy docs.

Reply:
I reverted https://github.com/mesosphere/dcos-docs-site/pull/4233/commits/64d470a2e5b000cc3bf9557fe6577a6e32c095a3, I'll go through and update the rest of the 2.2 docs to not use the `$VERSION` env var as well

Reply:
We should inform @cneth

Reply:
Sorry Casey for the back and forth!!

Reply:
Like I said, pick one.  But... what happens when we ship 2.2.1?   Doesn't that break all these pages?

Reply:
@cmastr no worries! Just want to get it figured out. 

@cneth yeah it would, for patch releases we have to bump versions somewhere, now its just in a lot more places (luckily find and replace is pretty easy).